pulsar-metadata/src/test/java/org/apache/pulsar/metadata/BaseMetadataStoreTest.java

```diff
@@ -19,23 +19,26 @@
 package org.apache.pulsar.metadata;
 
 import static org.testng.Assert.assertTrue;
-
 import java.util.concurrent.CompletionException;
-
+import org.apache.pulsar.tests.TestRetrySupport;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.DataProvider;
 
-public abstract class BaseMetadataStoreTest {
+public abstract class BaseMetadataStoreTest extends TestRetrySupport {
     protected TestZKServer zks;
 
     @BeforeClass(alwaysRun = true)
-    void setup() throws Exception {
+    @Override
+    protected void setup() throws Exception {
```

Comment:
Should we remove AfterClass/BeforeClass annotations?

Reply:
no, TestRetrySupport doesn't change that in any way.